docs/metrics/customizing-the-sdk/Program.cs

```diff
@@ -60,7 +60,7 @@ public static void Main(string[] args)
             // turn off the above default. i.e any
             // instrument which does not match any views
             // gets dropped.
-            // .AddView(instrumentName: "*", new DropAggregationConfig())
+            // .AddView(instrumentName: "*", new MetricStreamConfiguration() { Aggregation = Aggregation.Drop })
```

Comment:
Consider making a constant (e.g. `MetricStreamConfiguration.Drop`).

Reply:
yea. will work on it next step.
(This PR doesn't handle Aggregation part, only name change is covered)